doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-vscode-extension.adoc

```diff
@@ -13,8 +15,10 @@ For DMN or BPMN file support only, you can also search for the individual *DMN E
 . When the *Red Hat Business Automation Bundle* extension appears in VSCode, select it and click *Install*.
 . For optimal VSCode editor behavior, after the extension installation is complete, reload or close and re-launch your instance of VSCode.
 
-After you install the VSCode extension bundle, any `.dmn`, `.bpmn`, or `.bpmn2` files that you open in VSCode are automatically displayed as graphical models. Additionally, any `.scesim` files that you open are automatically displayed as tabular test scenario models for testing the functionality of your business decisions.
+After you install the VSCode extension bundle, any `.dmn`, `.bpmn`, or `.bpmn2` files that you open or create in VSCode are automatically displayed as graphical models. Additionally, any `.scesim` files that you open or create are automatically displayed as tabular test scenario models for testing the functionality of your business decisions.
 
 If the DMN, BPMN, or test scenario modelers open only the XML source of a DMN, BPMN, or test scenario file and displays an error message, review the reported errors and the model file to ensure that all elements are correctly defined.
 
 NOTE: For new DMN or BPMN models, you can also enter `dmn.new` or `bpmn.new` in a web browser to design your DMN or BPMN model in the online modeler. When you finish creating your model, you can click *Download* in the online modeler page to import your DMN or BPMN file into your {PRODUCT} project in VSCode.
+
+Before you can begin developing BPMN and DMN models in VSCode, you need to create a Maven project where you can build your assets and any other related resources for your application. For more information about creating Maven project, see {URL_GETTING_STARTED}[Getting started with {PRODUCT}].
```

Comment:
If you take my suggestion above to just remove this last bit and instead simply include the maven section in the two developing assemblies, then we're good. If not, then you need to conditionalize this for enterprise because the cross-ref link is incorrect when used for community. Meaning, just ifdef the whole paragraph since it doesn't apply to commnity. If you were to keep it for whatever, then just need to ifdef only the cross-ref as we typically do (for now).

Reply:
Done, I have removed this paragraph and included the maven project section in all four assemblies: Getting started guides (decision and process services) and developing guides (decision and process services). 

Thanks, Stetson